internal/infrastructure/rdb/place.go

```diff
@@ -572,6 +574,38 @@ func (p PlaceRepository) SaveGooglePlaceDetail(ctx context.Context, googlePlaceI
 	return nil
 }
 
+func (p PlaceRepository) SavePlacePhotos(ctx context.Context, userId string, placeId string, photoUrl string, width int, height int) error {
+	if err := runTransaction(ctx, p, func(ctx context.Context, tx *sql.Tx) error {
+		if ok, err := generated.PlacePhotos(generated.PlacePhotoWhere.PhotoURL.EQ(photoUrl)).Exists(ctx, tx); ok && err == nil {
+			// 画像のアドレスを示すURLがすでに保存済みの場合はスキップ
+			return nil
+		}
```

Comment:
goでは基本的に以下の記法でエラーハンドリングを行います
```go
err := doSomething()
if err != nil {
   return err
}
```

こうすることで、errを発生させる処理の直後でなければ
errはnilであるという前提で処理を書くことができます。

よって、以下のようにしたほうが良さそうです。
```suggestion

		ok, err := generated.PlacePhotos(generated.PlacePhotoWhere.PhotoURL.EQ(photoUrl)).Exists(ctx, tx); 
		if err != nil {
		return fmt.Errorf("error while checking place photo exists: %v", err)
		}
		if ok {
			// 画像のアドレスを示すURLがすでに保存済みの場合はスキップ
			return nil
		}
```

あと、すでに保存されていた場合は警告レベルくらいのログを出してあげたほうがわかりやすそうです。